osu.Game.Rulesets.Catch/Skinning/LegacyFruitPiece.cs

```diff
@@ -56,7 +57,7 @@ private void load(DrawableHitObject drawableObject, ISkinSource skin)
                 var hyperDash = new Sprite
                 {
                     Texture = skin.GetTexture(lookupName),
-                    Colour = Color4.Red,
+                    Colour = skin.GetHyperDashFruitColour()?.Value ?? Catcher.DefaultHyperDashColour,
```

Comment:
IMO the extension method should also handle the default.

Reply:
I wasn't sure at the beginning as I felt like it may not be good for readability (what value will this be fell back to if no setting provides a value), but sure.